feat: imported vars as override values (#423)
defenseunicorns/uds-cli: docs/overrides.md

              value:
                customAnnotation: "customValue"
```

> If [importing](../README.md#importingexporting-variables) a variable from another package, that variable can also be used to set a value, using the template syntax `${...}`. In the example below the `COLOR` variable is being used to set the `podinfo.ui.color` value.

Reviewer: ```suggestion
If using a variable that has been [exported](../README.md#importingexporting-variables) from another package, that variable can also be used to set a value, using the syntax `${...}`. In the example below the `COLOR` variable is being used to set the `podinfo.ui.color` value.
```

```
kind: UDSBundle
metadata:
  name: export-vars
  description: Example for using an imported variable to set an overrides value
  version: 0.0.1

packages:
  - name: output-var
    repository: localhost:888/output-var
    ref: 0.0.1
    exports:
      - name: COLOR

  - name: helm-overrides
    path: "../../packages/helm"
    ref: 0.0.1

>     imports:
>       - name: COLOR
>         package: output-var

Reviewer: ```suggestion
```

              value:
                customAnnotation: "customValue"
```

> If [importing](../README.md#importingexporting-variables) a variable from another package, that variable can also be used to set a value, using the template syntax `${...}`

Reviewer: Can we add an explicit example of what this looks like?

Author: added in latest commit.
